managed-configurations.html

```diff
@@ -1,240 +1 @@
ress Book Policy</li><li>Address List</li><li>Admin Audit Log Config</li><li>Availability Address Spaces</li><li>Availability Config</li><li>CAS Mailbox Plan</li><li>Client Access Rules</li><li>Distribution Groups<ul><li>Members</li></ul></li><li>DomainKeys Identified Mail Signing Config</li><li>Dynamic Distribution Groups</li><li>Email Address Policies</li><li>Global Address List</li><li>Inbound Connectors</li><li>Intra Organization Connectors</li><li>IRM Configuration</li><li>Journal Rules</li><li>Mail Flow<ul><li>Accepted Domains</li><li>Remote Domains</li></ul></li><li>Mailbox Plans</li><li>Mailboxes<ul><li>Permissions</li></ul></li><li>Malware Filter Policies<ul><li>Rules</li></ul></li><li>Mobile Device Access<ul><li>Device Access Rules</li></ul></li><li>Mobile Device Mailbox Policies</li><li>Modern Authentication</li><li>Offline Address Book</li><li>On-premises Organizations</li><li>Organization Config</li><li>Organization Relationship</li><li>Outbound Connectors</li><li>Outlook Web App Policies</li><li>Partner Applications</li><li>Policy Tip Config</li><li>Retention Policy</li><li>Retention Policy Tag</li><li>Role Assignment Policies</li><li>Sharing Policies</li><li>Tag External Emails</li><li>Transport Config</li><li>Transport Rules</li></ul></li><li>Security & Compliance<ul><li>Adaptive Scopes</li><li>Audit Configuration Policy</li><li>Case Hold Policies<ul><li>Rules</li></ul></li><li>Compliance Cases</li><li>Compliance Tags</li><li>DLP Compliance Policies<ul><li>Rules</li></ul></li><li>DLP Sensitive Information Types</li><li>File Plan Property Authorities</li><li>File Plan Property Categories</li><li>File Plan Property Citations</li><li>File Plan Property Departments</li><li>File Plan Property Reference Ids</li><li>File Plan Property Reference Sub Categories</li><li>Hosted Connection Filter Policies</li><li>Hosted Content Filter Policies<ul><li>Rules</li></ul></li><li>Hosted Outbound Spam Filter Policies<ul><li>Rules</li></ul></li><li>Information Governance<ul><li>Compliance Retention Event Types</li><li>Retention<ul><li>Rules</li></ul></li></ul></li><li>Label Policy</li><li>Labels</li><li>Protection Alerts</li><li>Threat Management<ul><li>Policy<ul><li>ATP Anti-phishing<ul><li>Rules</li></ul></li><li>ATP Safe Attachments<ul><li>Rules</li></ul></li><li>ATP Safe Links<ul><li>Rules</li></ul></li><li>Global Settings</li><li>Quarantine Policies</li></ul></li></ul></li></ul></li><li>Teams<ul><li>Apps<ul><li>Permission Policies</li><li>Setup Policies</li></ul></li><li>Meetings<ul><li>Live Event Policies</li><li>Meeting Policies</li><li>Meeting Settings</li></ul></li><li>Messaging Policies</li><li>Org-wide Settings<ul><li>Teams Settings</li></ul></li><li>Update Management Policies</li><li>Users<ul><li>External Access</li></ul></li><li>Voice<ul><li>Call Hold Policies</li><li>Calling Policies</li></ul></li></ul></li></ul><b>SharePoint</b><ul><li>Settings</li></ul><b>SharePoint Admin Center</b><ul><li>Settings and Policies</li><li>Sites</li></ul>
```

Comment:
This is kinda hard to review now. Why did we remove the lines between resources?